src/components/tokens/_tokens.scss

```diff
@@ -157,6 +148,15 @@
   --breadcrumb-text-hover: #192125;
   --block-quote-padding-x: 2rem;
   --block-quote-padding-y: 0.5rem;
+  --background-default: #ffffff;
```

Comment:
@mariannuar I just noticed that the final custom properties for the announcement tokens are missing the namespace.

Here is an example from the Accordion, `--accordion-color-body`.

These new tokens should start with `--announcement-`.

Reply:
That's right. I'm sorry, that's my bad 🥲 I'll change it